code/game/objects/structures/window.dm

```diff
@@ -675,6 +675,12 @@
 	dir = 5
 	icon_state = "rwindow_full"
 
+/obj/structure/window/reinforced/polarized/titanium
+	name = "electrochromic window"
+	desc = "Adjusts its tint with voltage. Might take a few good hits to shatter it."
+	basestate = "rwindow"
+	polarized = 1
+
```

Comment:
umm you didnt change anything or the material its the same as the default polarized window